datanode/entities/position.go

```diff
@@ -32,6 +32,10 @@ type positionSettlement interface {
 	TxHash() string
 }
 
+type positionState interface {
```

Comment:
Like lint says it looks like this is unused :)

Reply:
I thought I removed all of these, must've missed one